docs/usage/configuration/config_documentation.md

```diff
@@ -23,6 +23,14 @@ followed by a letter. Letters have the following meanings:
 For example, setting `redaction_retention_period: 5m` would remove redacted
 messages from the database after 5 minutes, rather than 5 months.
 
+In addition, configuration options referring to size use the following suffixes:
+
+* `M` = MB, or 1048576 bytes
+* `K` = KB, or 1024 bytes 
+
+For example, setting `max_avatar_size: 10M` means that Synapse will not accept files larger than 10MB
+for a user avatar. 
```

Comment:
I think this example might be clearer if we fully spell out what 10 Mebibytes means in this case? (10485760 bytes) Otherwise it kind of reads as "X is equal to X".